dev/com.ibm.ws.event.logging_fat/fat/src/com/ibm/ws/event/logging/FATTest.java

```diff
@@ -4,7 +4,7 @@
  * are made available under the terms of the Eclipse Public License 2.0
```

Comment:
@mrsaldana given the change IMO updating the copyright header date range to 2024 makes sense.